test/cli/harness/ipfs.go

```diff
@@ -78,3 +78,20 @@ func (n *Node) IPFSAdd(content io.Reader, args ...string) string {
 	log.Debugf("add result: %q", out)
 	return out
 }
+
+func (n *Node) IPFSDagImport(content io.Reader, args ...string) string {
+	log.Debugf("node %d dag import with args: %v", n.ID, args)
+	fullArgs := []string{"dag", "import"}
```

Comment:
nit: this makes sure we don't over-fetch when we use a third-party CAR fixtures from some IPIP that does not have all the blocks for listed root CID.
```suggestion
	fullArgs := []string{"dag", "import", "--pin=false"}
```

Reply:
Hmmm... if we do this, then the command doesn't print the CID. We have to remove the response of the command then.

Reply:
Perhaps we could assert by doing `ipfs block stat --offline`  instead?